scipy/special/_add_newdocs.py

```diff
@@ -1847,19 +1847,30 @@ def add_newdoc(name, doc):
     """)
 
 add_newdoc("chndtr",
-    """
+    r"""
     chndtr(x, df, nc, out=None)
 
     Non-central chi square cumulative distribution function
 
+    The cumulative distribution function is given by:
+
+    .. math::
+
+        P(\chi^{\prime 2} \vert \nu, \lambda) =\sum_{j=0}^{\infty}
+        e^{-\lambda /2}
+        \frac{(\lambda /2)^j}{j!} P(\chi^{\prime 2} \vert \nu + 2j),
+
+    where :math:`\nu > 0` is the degrees of freedom and :math:`\lambda \geq 0`
+    is the non-centrality parameter.
```

Comment:
Would you mind connecting the parameter names here to what is used in the routine? If I understand correctly `$\mu$` corresponds to `df` and `$\lambda$` corresponds to `nc`.